key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
no default private key, so people setting this up are forced to generate their own instead of keeping this insecure one. There is no readme (yet) so it's trivial to miss it as well.

Reply:
An accident.